src/chart/generateCategoricalChart.tsx

```diff
@@ -721,6 +788,10 @@ const calculateOffset = (
   };
 };
 
+type AxisMap = {
+  [k: string]: BaseAxisProps;
+};
```

Comment:
I wonder whether this is correct. Does the axis map not hold all the props for each axis (i.e. cartesian and polar)?